experimental/README.md

```diff
@@ -4,4 +4,5 @@
 
 | Template | Description |
 | --- | --- |
-| [`ner_spancat`](ner_spancat) | Example SpanCategorizer project using Indonesian NER |
\ No newline at end of file
+| [`ner_spancat`](ner_spancat) | Example SpanCategorizer project using Indonesian NER |
+| [`healthsea_spancat`](healthsea_spancat) | Healthsea project comparing performance between NER and Spancat architecture |
```

Comment:
I'm not sure what "Healthsea project" would mean, maybe it makes more sense to call this "Project comparing performance between NER and Spancat architecture on Healthsea data", and then have a linkout for "Healthsea"?